.github/workflows/ci_dockers.yml

```diff
@@ -14,8 +14,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python_version: [3.7, 3.8]
-        pytorch_version: [1.5]
+        python_version: [3.6, 3.7, 3.8]
+        pytorch_version: [1.3, 1.4, 1.5, 1.6, 1.7]
```

Comment:
```suggestion
        pytorch_version: [1.3, 1.5, 1.7]
```

Reply:
Not testing all supported versions?

Reply:
we are testing it with each in regular CI, this is doing just installation, so no need for so dense testing just edge-cases

Reply:
Alright.